### ISSUE TITLE
action 修改 dts

### ISSUE THREAD

**Issue by lbbboy** [hzyitc/armbian-onecloud#4]
```
对于V1.0的板子，您可以直接把dts中usb0的dr_mode改成host来启动该USB。
```
怎么用脚本实现呢？

**Comment by soulant** [comment]
```
cd ./armbian-onecloud/patch/kernel/archive/meson-5.10
cp support-xunlei-onecloud.patch{,bak}
sed -i "/dr_mode/ s/otg/host/" support-xunlei-onecloud.patch
```
To build at the same time, you must write another configuration and modify the Makefile.

**Comment by hzyitc** [comment]
Thank @soulant.

Note: The path of `edge` is `meson-5.18`  instand of `meson-5.10`.

---

感谢 @soulant 。

注意：`edge`的路径是`meson-5.18`而不是`meson-5.10`。

**Comment by hzyitc** [comment]
Close due to fix in [6f25cd6](https://github.com/hzyitc/armbian-onecloud/commit/6f25cd62f2cfe43ab41ce0891c2bfd53c83baac2)
